pt/endpoints/address/model.py

```diff
@@ -1,4 +1,6 @@
-import uuid
+# import uuid
```

Comment:
We can remove this line instead of commenting it if we would not use this package anymore.

Reply:
Fix in 2d5b37.